best-practices.md

```diff
@@ -641,6 +641,10 @@ a number of the common official relations that are used in production STAC imple
 | next      | Indicates that the link's context is a part of a series, and that the next in the series is the link target. Typically used in STAC by API's, to return smaller groups of Items or Catalogs/Collections. |
 | preview   | Refers to a resource that serves as a preview (see [RFC 6903, sec. 3](https://tools.ietf.org/html/rfc6903#section-3)), usually a lower resolution thumbnail. In STAC this would usually be the same URL as the [thumbnail](#thumbnail) asset, but adding it as a link in addition enables OGC API clients that can't read assets to make use of it. It also adds support for thumbnails to STAC Catalogs as they can't list assets. |
 
+Being liberal with the `links` also means that it's ok to have repeated links with the same `rel`. For example the
```

Comment:
Was that meant to say href instead of rel?

Reply:
In the example here it should be href, but good to point out that same `rel` types are also ok

Reply:
Yes, will fix.